src/rust/catloop/mod.rs

```diff
@@ -334,7 +334,9 @@ impl CatloopLibOS {
                 }
             }
         }
-        self.runtime.free_queue::<CatloopQueue>(qd);
+        self.runtime
+            .free_queue::<CatloopQueue>(&qd)
+            .expect("queue should exist");
```

Comment:
To follow our guidelines, could you add a comment here explaining why it is safe to call `expect()` here?